src/ext/graphql/transmodelapi/schema.graphql

```diff
@@ -1205,10 +1205,24 @@ type ValidityPeriod {
 type ViaTrip {
   "A list of routing errors, and fields which caused them"
   routingErrors: [RoutingError!]!
-  "A list of lists of which indices of the next segment the trip pattern can be combined with"
-  tripPatternCombinations: [[[Int!]!]!]!
-  "A list of lists of the trip patterns for each segment of the journey"
-  tripPatterns: [[TripPattern!]!]!
+  "A list of the acceptable combinations of the trip patterns in this segment and the next segment."
+  tripPatternCombinations: [ViaTripPatternPair!]!
+  "A list of segments of the via search. The first segment is from the start location to the first entry in the locations list and the last is from the last entry in the locations list to the end location."
+  tripPatternsPerSegment: [ViaTripPatternSegment!]!
+}
+
+"An acceptable combination of trip patterns between two segments of the via search"
+type ViaTripPatternPair {
```

Comment:
Is there a better name for this like `ViaTripPatternConnection` or `ViaConnection` ? I assume there is noting in the Transmodel for this?

Reply:
The best thing I can come up with is `ViaConnection`, `ViaLocationConnection` or `ViaTripConnection`. I vote for `ViaConnection` it allows us to add information like walking directions, spare time, and so on later.

Reply:
Renamed to `ViaConnection`